packages/blade/src/utils/useBreakpoint/useBreakpoint.web.test.ts

```diff
@@ -4,7 +4,7 @@ import { useBreakpoint } from './';
 import { breakpoints } from '~tokens/global';
 
 describe('useBreakpoint', () => {
-  it('should detect mobile and return smallest breakpoint of mobile devices', () => {
+  it('should return default breakpoint and device', () => {
```

Comment:
its not returning default right? we're returning mobile and base

Reply:
oh yes this changed multiple times. will update